lib/pluck_each.rb

```diff
@@ -16,10 +16,13 @@ def pluck_each(*args)
 
     def pluck_in_batches(*column_names)
       options = column_names.extract_options!
+      string_column_names = column_names.map(&:to_s)
```

Comment:
I put it in a different field `string_column_names` because we still need to know if the original request was for a single field; we could capture that as a boolean but did not figure it was that important